### PR TITLE
CORDA-1521 - Fix rpc attachment smoke test / better AMQP logging

### PR DESCRIPTION
**Pull request by fenryka**

### CODE REVIEW

File: serialization/src/main/kotlin/net/corda/serialization/internal/amqp/SerializerFactory.kt [hunk 1]
```diff
@@ -107,12 +111,17 @@ open class SerializerFactory(
                     makeMapSerializer(declaredTypeAmended)
                 }
             }
-            Enum::class.java.isAssignableFrom(actualClass
-                    ?: declaredClass) -> serializersByType.computeIfAbsent(actualClass ?: declaredClass) {
-                whitelist.requireWhitelisted(actualType)
-                EnumSerializer(actualType, actualClass ?: declaredClass, this)
+            Enum::class.java.isAssignableFrom(actualClass ?: declaredClass) -> {
+                logger.debug("class=[${actualClass} | $declaredClass] is an enumeration type=${declaredType} "
```
[the remark this answers is not in the document]
Comment (fenryka) [reply serialization/src/main/kotlin/net/corda/serialization/internal/amqp/SerializerFactory.kt]: done

File: serialization/src/main/kotlin/net/corda/serialization/internal/amqp/SerializerFactory.kt [hunk 2]
```diff
@@ -107,12 +111,17 @@ open class SerializerFactory(
...
+                        + "isEnum=${declaredType::class.java.isEnum}")
+
+                serializersByType.computeIfAbsent(actualClass ?: declaredClass) {
+                    whitelist.requireWhitelisted(actualType)
+                    EnumSerializer(actualType, actualClass ?: declaredClass, this)
+                }
             }
             else -> {
+                logger.debug("class=[${actualClass} | $declaredClass] is a composite type")
```
[the remark this answers is not in the document]
Comment (fenryka) [reply serialization/src/main/kotlin/net/corda/serialization/internal/amqp/SerializerFactory.kt]: done

File: serialization/src/main/kotlin/net/corda/serialization/internal/amqp/SerializerFactory.kt [hunk 3]
```diff
@@ -198,6 +207,7 @@ open class SerializerFactory(
     @Throws(NotSerializableException::class)
     fun get(typeDescriptor: Any, schema: SerializationSchemas): AMQPSerializer<Any> {
         return serializersByDescriptor[typeDescriptor] ?: {
+            logger.trace("get Serializer descriptor=${typeDescriptor}")
```
[the remark this answers is not in the document]
Comment (anthonykeenan) [reply serialization/src/main/kotlin/net/corda/serialization/internal/amqp/SerializerFactory.kt]: Oh no it isn't

**Comment by anthonykeenan** [comment]
LGTM apart from that one set of curly brackets that didn't get removed

**Pull request merged**